services/QuillLMS/client/app/bundles/Connect/components/studentLessons/lesson.jsx

```diff
@@ -369,6 +370,7 @@ export class Lesson extends React.Component {
   }
 
   saveSessionIdToState = () => {
```

Comment:
It feels like we might want to rename this method since it's doing more now? Like `saveSessionDataToState`?

Reply:
I think the name change makes sense-- I'll update